package.json

```diff
@@ -148,6 +148,7 @@
     "react-addons-test-utils": "0.14.3",
     "react-hot-loader": "1.3.0",
     "rewire": "2.3.3",
+    "semver": "5.1.0",
```

Comment:
I feel like we should put this in `devDependencies`, but I'm not sure if that would break production deployment or not (`npm install` does not install dev deps when `NODE_ENV=production`).